src/CcdImage.cc

```diff
@@ -130,6 +130,20 @@ CcdImage::CcdImage(afw::table::SourceCatalog &catalog, std::shared_ptr<lsst::afw
     }
 }
 
+std::pair<int, int> CcdImage::countStars() const {
+    int measuredStars = 0;
+    int refStars = 0;
+    for (auto &measuredStar : _catalogForFit) {
```

Comment:
please insert a `const` after `auto` since the code does not modify `_catalogForFit`